source/clear-linux/reference/bundles/bundles.rst

```diff
@@ -6,16 +6,6 @@ Available bundles
 This document provides a current list of `available bundles`_ as 
 of ``12 September 2018``.                                      
 
-See in depth descriptions of the following bundles:
-
-.. toctree::
-   :maxdepth: 1
-
-   containers-basic
-   openssh-server
-   os-core
-   kvm-host
-
 Bundle list
 ===========
```

Comment:
Per our standard heading levels, first level heading after page title should be updated to use "*"